api/serializers/student.py

```diff
@@ -13,6 +13,10 @@
 
 
 class StudentWriteSerializer(serializers.ModelSerializer[Student]):
+    # For Student, we need Student.id (and not just personal_info.id) so we can pass it
```

Comment:
As I understand Student.id is same as personal_info.id. Student doesn't have his own id.